.gitignore

```diff
@@ -17,7 +17,7 @@ proj.pro.user
 /src/packages
 .qmake.stash
 /.vs
-/.vs
```

Comment:
When opening the project in Visual Studio, we need to ignore `.vs` path.

Reply:
yeah but it was duplicated like
.vs
.vs <- removed it